src/jit/codegenxarch.cpp

```diff
@@ -3563,7 +3579,7 @@ void CodeGen::genStructPutArgUnroll(GenTreePutArgStk* putArgNode, unsigned baseV
     noway_assert(putArgNode->TypeGet() == TYP_STRUCT);
```

Comment:
For SIMD types we directly store them in genPutStructArgStk() so we don't need to handle them here. I will add a comment.